ingestion-beam/src/main/java/com/mozilla/telemetry/io/Read.java

```diff
@@ -92,32 +118,33 @@ public BigQueryInput(ValueProvider<String> tableSpec, BigQueryReadMethod method,
 
     @Override
     public Result<PCollection<PubsubMessage>> expand(PBegin input) {
-      BigQueryIO.TypedRead<PubsubMessage> read = BigQueryIO
-          .read((SchemaAndRecord schemaAndRecord) -> {
-            TableSchema tableSchema = schemaAndRecord.getTableSchema();
-            GenericRecord record = schemaAndRecord.getRecord();
-            byte[] payload = ((ByteBuffer) record.get("payload")).array();
-
-            // We copy only string-type fields to attributes, which is complete for raw and
-            // error tables; decoded payload tables have the nested metadata object also encoded in
-            // the payload, so we can safely drop the metadata object here and rely on ParsePayload
-            // to parse attributes from it.
-            Map<String, String> attributes = new HashMap<>();
-            tableSchema.getFields().stream() //
-                .filter(f -> "STRING".equals(f.getType())) //
-                .forEach(f -> attributes.put(f.getName(), (String) record.get(f.getName())));
-            return new PubsubMessage(payload, attributes);
-          }) //
+      BigQueryIO.TypedRead<PubsubMessage> read = BigQueryIO.read((SchemaAndRecord schemaAndRecord) -> {
+        TableSchema tableSchema = schemaAndRecord.getTableSchema();
+        GenericRecord record = schemaAndRecord.getRecord();
+        byte[] payload = ((ByteBuffer) record.get("payload")).array();
+
+        // We copy only string-type fields to attributes, which is complete for raw and
+        // error tables; decoded payload tables have the nested metadata object also
+        // encoded in
+        // the payload, so we can safely drop the metadata object here and rely on
+        // ParsePayload
+        // to parse attributes from it.
+        Map<String, String> attributes = new HashMap<>();
+        tableSchema.getFields().stream() //
+            .filter(f -> "STRING".equals(f.getType())) //
+            .forEach(f -> attributes.put(f.getName(), (String) record.get(f.getName())));
+        return new PubsubMessage(payload, attributes);
+      }) //
           .withCoder(PubsubMessageWithAttributesCoder.of()) //
           .withTemplateCompatibility() //
           .withMethod(method.method);
       switch (source) {
-        case TABLE:
-          read = read.from(tableSpec);
-          break;
-        default:
-        case QUERY:
-          read = read.fromQuery(tableSpec).usingStandardSql();
+      case TABLE:
+        read = read.from(tableSpec);
+        break;
+      default:
+      case QUERY:
+        read = read.fromQuery(tableSpec).usingStandardSql();
```

Comment:
It looks like you have editor settings that are fighting with what exists here. In the end, you can call `mvn spotless:apply` and it'll auto-format everything back to what google java style wants anyway.